.travis.yml

```diff
@@ -2,18 +2,23 @@ if: (branch = master) OR (type = pull_request) OR (tag IS present)
 matrix:
   include:
     - language: rust
-      rust: 1.33.0
+      rust: 1.36.0
       cache: 
         cargo: true
         directories:
           - driver/target
+          - listener/target
       before_script:
         - rustup component add clippy
       script:
         - cd driver/
         - cargo clippy -- -D warnings
```

Comment:
Over here, you get the same issue, which I am having in the other PR:
```
error: error reading Clippy's configuration file `/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/mongodb-0.3.12/clippy.toml`: found deprecated field `cyclomatic-complexity-threshold`. Please use `cognitive-complexity-threshold` instead.
    Checking crossbeam v0.6.0
    Checking tokio-reactor v0.1.8
error: aborting due to previous error
```
I could not yet find a solution for that :(